data_model/src/isi.rs

```diff
@@ -100,8 +100,6 @@ pub struct SetParameterBox {
     /// The configuration parameter being changed.
     #[serde(flatten)]
     pub parameter: EvaluatesTo<Parameter>,
-    /// The entity requesting the change.
-    pub source_id: EvaluatesTo<IdBox>,
```

Comment:
is this ok?

Reply:
I do think that if we need it, we should reintroduce it **when we need it**.

Reply:
Yes, I didn't understand the reason we introduced this field. To me it looks redundant because we always know the authority, who submits transaction